iam-login-service/src/main/java/it/infn/mw/iam/api/scim/provisioning/ScimUserProvisioning.java

```diff
@@ -174,12 +171,12 @@ private ArrayList<String> parseFilters(final String filtersParameter) {
   }
 
 
-  private boolean filterEvaluation(ArrayList<String> parsedFilters){
-    if(parsedFilters.size()==3){
-      for(Attributes attribute : Attributes.values()){  
-        if(attribute.toString().equalsIgnoreCase(parsedFilters.get(0))){
-          for(AttributeOperators operator : AttributeOperators.values()){
-            if(operator.toString().equalsIgnoreCase(parsedFilters.get(1))){
+  private boolean filterEvaluation(ArrayList<String> parsedFilters) {
+    if (parsedFilters.size() == 3) {
```

Comment:
can we reduce the overall nesting? maybe with an early return if parsedFilters.size() is not 3?